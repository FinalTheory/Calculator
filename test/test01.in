exp( ( 3.14 + sin( 2.67 ) ) / 5. * .50 * x )
( 1.28 + 2.32 ) * ln( 3. ) * x
sqrt( 2 ^ 2.3 + 3 * 4 ^ .5 ) + 4
( 1 + 3 ) * 6.5 / 4.2 ^ 2.3
1 + 2
65 * 2 + 398.678
1 / ( 1 - 2 )
( ( ( ( 1 + 2 ) * 3 ) + 4 ) * 5 + 128 ) / 64.2
( 3.4 ^ x ) / 256.7
x / ( 4 + x ^ 2 )
x / ( 4 + x ) ^ 2
( (-) 3 + 2.5 ) / x
6 - (-) 3 ^ 3 * x