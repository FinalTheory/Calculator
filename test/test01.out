1.5841
5.0616
7.3052
0.9583
3.0000
528.6780
-1.
3.0062
0.0187
0.2270
0.0459